packages/opencensus-core/src/stats/view.ts

```diff
@@ -221,18 +230,40 @@ export class BaseView implements View {
    */
   private toPoint(timestamp: Timestamp, data: AggregationData): Point {
     let value;
-
     if (data.type === AggregationType.DISTRIBUTION) {
-      // TODO: Add examplar transition
-      const {count, sum, sumOfSquaredDeviation} = data;
+      const {count, sum, sumOfSquaredDeviation, exemplars} = data;
+      const buckets = [];
+      for (let bucket = 0; bucket < data.bucketCounts.length; bucket++) {
+        const bucketCount = data.bucketCounts[bucket];
+        let statsExemplar;
+        if (exemplars) {
+          statsExemplar = exemplars[bucket];
+        }
+
+        let metricBucket;
+        if (statsExemplar) {
+          // Bucket with an Exemplar.
+          metricBucket = {
+            count: bucketCount,
+            exemplar: {
+              value: statsExemplar.value,
+              timestamp: timestampFromMillis(statsExemplar.timestamp),
+              attachments: statsExemplar.attachments
+            }
+          } as metricBucket;
```

Comment:
Optional: rather than use a type cast here with a `let` above, could you make a helper function say `getMetricBucket` that returns a `metricBucket` type and then you could even just call `buckets.push(getMetricBucket(statsExemplar, bucketCount))` with no intermediate variable?

Reply:
Good suggestion, done.